docs/configuration/index.md

```diff
@@ -628,6 +628,30 @@ As a result, invoking `/packit test` will be automatically interpreted as `/pack
 `default_identifier` and `default_labels` are not supposed to be used simultaneously. You should configure
 only one of these options.
 :::
+
+#### parse_time_macros
+
+(*dict*) Macros to be explicitly defined or undefined at spec file parse time.
+To undefine a macro, set its value to `null`.
+This can be useful for instance when the spec file is OS-specific and Packit runs on a different OS.
```

Comment:
> Packit runs on a different OS.

We could mention here what Packit runs on exactly.

Reply:
You mean Packit Service, right? I was thinking about that, let me add it (but we should keep in mind to update it when we upgrade to c10s :slightly_smiling_face:).